docs/src/pages/components/tables/EnhancedTable.tsx

```diff
@@ -19,6 +19,7 @@ import FormControlLabel from '@material-ui/core/FormControlLabel';
 import Switch from '@material-ui/core/Switch';
 import DeleteIcon from '@material-ui/icons/Delete';
 import FilterListIcon from '@material-ui/icons/FilterList';
+import { visuallyHidden } from '@material-ui/lab/VisuallyHidden';
```

Comment:
In this demo, I think that we could use 

```suggestion
import VisuallyHidden from '@material-ui/lab/VisuallyHidden';
```

My concern with the rating was performance, but here, It feels like a component is more idiomatic. What do you think?

Reply:
It's used in both on an `input` element, as well as on `span`. For the scenario of `span` it could be used, but for the inputs we will need to add additional element, and probably convert the `span` to `div`? Wasn't sure if we are ok with chaning the strucutre. What do you think?

Reply:
Ok, sounds fair